### ISSUE TITLE
Update Product Names in the Repo i.e `Chispend` to `Unispend` 

### ISSUE THREAD

**Issue by phyleria** [Chimoney/chimoney-community-projects#267]
- [ ] Update all occurrences of `Chispend` in the repository's code, documentation, and files to `Unispend` and `Chiconnect` to `Chimoney Developer API`

- [ ] Ensure that the updates are accurately reflected and do not break any existing functionality.

**Comment by pranavjainjs** [comment]
Hi, I'd like to work on this issue.

**Comment by ayan-joshi** [comment]
@phyleria Thanks assign me I'll work on it

**Comment by HarshitVashisht11** [comment]
please assign me this issue i can fix this issue

**Comment by ayan-joshi** [comment]
@phyleria I also have to change chiconnect to Chimoney Developer API ? 
This are there in code like package.json also there are some folders name as well as Chincoonect so I have change them as well?

**Comment by phyleria** [comment]
Yeah change the folders too

**Comment by AnanyaJindal1145** [comment]
Hi! Can I be assigned this issue?

**Comment by phyleria** [comment]
Hi @AnanyaJindal1145 

The issue has already been assigned.

**Comment by phyleria** [comment]
Hi @ayan-joshi 

Will you proceed with updating the rest of the names?

**Comment by ayan-joshi** [comment]
> Hi @ayan-joshi
> 
> Will you proceed with updating the rest of the names?

Yes , but it's getting some what difficult as there are folders name with it but I'll do it and commit to my pr

**Comment by satwikrprabhu** [comment]
Hey is there anything i can do?

**Comment by Rishpwl** [comment]
hey can i assigned with this issue?

**Comment by ayan-joshi** [comment]
It's completed I created the pr 😊
